<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityWizardNameAndTypePage</class>
 <widget class="QWizardPage" name="ActivityWizardNameAndTypePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>485</height>
   </rect>
  </property>
  <property name="title">
   <string>Activity Name and Type</string>
  </property>
  <property name="subTitle">
   <string>Fill in activity name and select its type.</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="buddy">
      <cstring>typeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="typeComboBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="overlapCheckBox">
     <property name="text">
      <string>Can overlap with other activities</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="moreThanOnceCheckBox">
     <property name="text">
      <string>Ativity happens more than once or during period of time</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Repeats on</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="weekDaysRadioButton">
        <property name="text">
         <string>Week days</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="specificDaysRadioButton">
        <property name="text">
         <string>Specific days</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>overlapCheckBox</tabstop>
  <tabstop>moreThanOnceCheckBox</tabstop>
  <tabstop>weekDaysRadioButton</tabstop>
  <tabstop>specificDaysRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
